ecs/sdk.go

```
@@ -264,9 +264,18 @@ func (s sdk) CreateChangeSet(ctx context.Context, name string, template []byte)
 		return "", err
 	}
 
-	err = s.CF.WaitUntilChangeSetCreateCompleteWithContext(ctx, &cloudformation.DescribeChangeSetInput{
+	s.CF.WaitUntilChangeSetCreateCompleteWithContext(ctx, &cloudformation.DescribeChangeSetInput{ // nolint:errcheck
```

Is there no other relevant error that can be returned here?

---

we actually get a false-negative error here. This API returns "ResourceNotReady" error as the changeset failed to  create, but we still need to wait for changeset creation completion before we can inspect it's state

---

Would be good to add a comment about why we're ignoring the error then :)

---

done